src/ConnectedVMware/docs/New-AzConnectedVMwareVM.md

```diff
@@ -47,27 +47,44 @@ Please note some properties can be set only during virtual machine instance crea
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Create Virtual Machine on the given Resource Pool
 ```powershell
-{{ Add code here }}
+New-AzConnectedVMwareVM -Name "test-vm" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" -Location "eastus" -ExtendedLocationName "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourcegroups/test-rg/providers/microsoft.extendedlocation/customlocations/azcli-test-cl" -ExtendedLocationType "CustomLocation" -VCenterId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/test-rg/providers/Microsoft.ConnectedVMwarevSphere/VCenters/azcli-test-vc" -PlacementProfileResourcePoolId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/test-rg/providers/Microsoft.ConnectedVMwarevSphere/resourcepools/test-rp" -TemplateId "/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/test-rg/providers/Microsoft.ConnectedVMwarevSphere/virtualMachineTemplates/test-vmtmpl"
```

Comment:
>-Name "test-vm" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" [](http://example.com/codeflow?start=24&length=99)

these params are not defined in the usage above? Does the cmdlet accept these? I think since swagger has resourceUri only, the auto generated cmdlet will only accept that. We might need to add a custom cmdlet for the params we desire to use which are not there in the swagger.

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/stable/2023-10-01/connectedvmware.json#L3252](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/stable/2023-10-01/connectedvmware.json#L3252)

Reply:
updated the docs.